.github/workflows/make_wheel_Windows.sh

```diff
@@ -6,8 +6,9 @@ export BAZEL_VC="C:/Program Files (x86)/Microsoft Visual Studio/2019/Enterprise/
 python -m pip install --default-timeout=1000 wheel setuptools tensorflow==$TF_VERSION
 bash ./tools/testing/build_and_run_tests.sh
 
+python configure.py
+
 bazel.exe build \
-  -c opt \
```

Comment:
Why `-c opt` is removed?

Reply:
It's written by default in configure:
https://github.com/tensorflow/addons/blob/master/configure.py#L108

Just wanted to set that as the central source of build args